x-pack/plugins/ml/public/explorer/_explorer.scss

```diff
@@ -40,6 +40,12 @@
   .results-container {
     padding: $euiSize 0px $euiSize 0px;
 
+    .mlAnomalyExplorer__filterBar {
```

Comment:
It's probably safe to remove this from this descendent tree because the className is specific enough and uses the `ml` prefix.

Reply:
Good call - updated in 7032a0616ed061e43586af7f446ca00bbf237d5c